src/pages/People.tsx

```diff
@@ -0,0 +1,155 @@
+    }
+
+    return undefined;
+  };
+
+  useEffect(() => {
+    const fetchData = async () => {
+      setIsLoading(true);
+
+      try {
+        const res = await getPeople();
+
+        if (res.length) {
+          setPeople(res);
+        } else {
+          setError(ERROR.EMPTYARRAY);
+        }
+      } catch (err) {
+        setError(ERROR.OTHERERROR);
+      } finally {
+        setIsLoading(false);
+      }
+    };
+
+    fetchData();
+  }, []);
+
+  return (
+    <>
+      <div className="block">
+        <div className="box table-container">
+
+          {isLoading ? (
+            <Loader />
+          ) : (
+            <table
+              data-cy="peopleTable"
+              className="table is-striped is-hoverable is-narrow is-fullwidth"
+            >
+              {error === ERROR.OTHERERROR && (
+                <p data-cy="peopleLoadingError" className="has-text-danger">
+                  Something went wrong
+                </p>
+              )}
+              <thead>
+                <tr>
+                  <th>Name</th>
+                  <th>Sex</th>
+                  <th>Born</th>
+                  <th>Died</th>
+                  <th>Mother</th>
+                  <th>Father</th>
+                </tr>
+              </thead>
+
+              <tbody>
+
+                {error === ERROR.EMPTYARRAY && (
+                  <p data-cy="noPeopleMessage">
+                    There are no people on the server
+                  </p>
+                )}
+
+                {people.map(person => (
+                  <tr
+                    data-cy="person"
+                    key={person.slug}
+                    className={classNames({
+                      'has-background-warning': selectedPerson === person.slug,
+                    })}
+                  >
+                    <td>
+                      <Link
+                        to={`${person.slug}`}
+                        className={classNames({
+                          'has-text-danger': person.sex === 'f',
+                        })}
+                      >
+                        {person.name}
+                      </Link>
+                    </td>
+
+                    <td>{person.sex}</td>
+                    <td>{person.born}</td>
+                    <td>{person.died}</td>
+
+                    {findMother(person) ? (
+                      <td>
+                        <Link
+                          to={`${findMother(person)?.slug}`}
+                          className="has-text-danger"
+                        >
+                          {person.motherName}
+                        </Link>
+                      </td>
+                    ) : (
+                      <td>{person.motherName || '-'}</td>
+                    )}
+
+                    {findFather(person) ? (
+                      <td>
+                        <Link
+                          to={`${findFather(person)?.slug}`}
```

Comment:
You are calling the findFather function twice - once for the condition and once for the Link 'to' prop. This is inefficient. Instead, call it once and store the result in a variable to be used in both places.